test/test.spec.ts

```diff
@@ -94,6 +95,20 @@ describe('service', function () {
       // Assert
       expect(componentInstance).toEqual(componentInstance2);
     })
+
```

Comment:
So I assume leaving the test as is above is OK for now? It's unlikely we will do any significant refactoring of tests for our feature.